tools/include/edm4hep2json.hxx

```diff
@@ -0,0 +1,173 @@
+#ifndef EDM4HEP_TO_JSON_H__
+#define EDM4HEP_TO_JSON_H__
+
+// event data model
+#include "edm4hep/MCParticleCollection.h"
+#include "edm4hep/VertexCollection.h"
+#include "edm4hep/TrackCollection.h"
+#include "edm4hep/ClusterCollection.h"
+#include "edm4hep/CalorimeterHitCollection.h"
+#include "edm4hep/ReconstructedParticleCollection.h"
+
+// podio specific includes
+#include "podio/EventStore.h"
+
+// JSON
+#include "nlohmann/json.hpp"
+
+// STL
+#include <vector>
+#include <iostream>
+#include <fstream>
+#include <exception>
+#include <cassert>
+
+
+nlohmann::json processEvent(podio::EventStore& store,
```

Comment:
```suggestion
template<typename PodioStoreT>
nlohmann::json processEvent(const PodioStoreT& store,
```
This should automatically make it also work with the `Frame`.

Reply:
Thanks for the suggestion